### PR TITLE
DOCSP-30047 | Added patch version 1.10.1

### PR DESCRIPTION
**Pull request by caitlindavey**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-kafka-connector/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-30047>
Staging - <https://docs-mongodbcom-staging.corp.mongodb.com/kafka-connector/docsworker-xlarge/DOCSP-30047-1.10.1Patch/>

## Self-Review Checklist

- [X] Is this free of any warnings or errors in the RST?
- [X] Did you run a spell-check?
- [ ] Did you run a grammar-check?
- [ ] Are all the links working?

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "esbonio.sphinx.confDir": ""
+}
```
Comment (mongoKart) [inline .vscode/settings.json:3]: This is from your directory of VS Code settings and shouldn't be committed. You should be able to add the .vscode directory to your gitignore file

File: source/whats-new.txt [hunk 2]
```diff
@@ -12,6 +12,7 @@ What's New
 
 Learn what's new by version:
 
+* :ref:`Version 1.10 <kafka-connector-whats-new-1.10.1>`
```
Comment (mongoKart) [inline source/whats-new.txt:15]:
```suggestion
* :ref:`Version 1.10.1 <kafka-connector-whats-new-1.10.1>`
```

File: source/whats-new.txt [hunk 3]
```diff
@@ -27,6 +28,20 @@ Learn what's new by version:
...
+--------------------
+
+.. important:: Upgrade to Version 1.10.1
+  
+  Version 1.9 introduced a bug that can cause a resource leak on the connector-side 
+  and the server-side related to ``MongoSourceTask.start``. 
+
+  Upgrade to version 1.10.1 if you are using version 1.9 or 1.10 of the connector.
+
+- Fixed a resource leak related to ``MongoSourceTask.start`` that was introduced in 1.9.0.
```
Comment (mongoKart) [inline source/whats-new.txt:43]: I: Good to be consistent with versions. Previously we just call this "1.9"

```suggestion
- Fixed a resource leak related to ``MongoSourceTask.start`` that was introduced in version 1.9. 
```

File: source/whats-new.txt [hunk 4]
```diff
@@ -27,6 +28,20 @@ Learn what's new by version:
...
 
+.. _kafka-connector-whats-new-1.10.1:
+
+What's New in 1.10.1
+--------------------
+
+.. important:: Upgrade to Version 1.10.1
+  
+  Version 1.9 introduced a bug that can cause a resource leak on the connector-side 
+  and the server-side related to ``MongoSourceTask.start``.
```
Comment (mongoKart) [inline source/whats-new.txt:39]: Suggestion for clarity (and extraneous hyphenation):

```suggestion
  Version 1.9 introduced a bug related to ``MongoSourceTask.start`` that can cause a resource leak on 
  both the connector side and the server side . 
```

**Review by mongoKart — CHANGES_REQUESTED** [review]
A few small things we didn't catch the first time...

File: .vscode/settings.json (hunk 1 above)
Comment (caitlindavey) [reply .vscode/settings.json:3]: Should I remove this from the PR?

**Review by mongoKart — APPROVED** [review]
LGTM!

**Pull request merged**